apis/userapi/userapi.yaml

```diff
@@ -168,6 +168,81 @@ paths:
         AllowDynamicQueryParameters: false
         AllowDynamicFormParameters: false
         IsMultiContentStreaming: false
+  /user/v3/create:
+    post:
+      description: >-
+        This is an Enhanced API with good throughput, is used to create a user
```

Comment:
This api is lightweight of v2. This api will create user with very minimum attribute (firstName, email/phone, password, emailVerified/phoneVerified). All user created by this endpoint will be linked with custodian org.

Reply:
done